onnxscript/function_libs/torch_lib/ops/core.py

```diff
@@ -6583,10 +6583,11 @@ def aten_prelu_backward(
     raise NotImplementedError()
 
 
-def aten_prod(self: TensorType, dtype: Optional[int] = None) -> TensorType:
+@torch_op(("aten::prod", "aten::prod.dim_int"), trace_only=True)
```

Comment:
```suggestion
@torch_op("aten::prod.dim_int", trace_only=True)
```

this should be good enough, since prod does not have a `.default` overload